admin/sql/create_tables.sql

```diff
@@ -207,6 +207,15 @@ CREATE TABLE user_relationship (
     created             TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
 );
 
+CREATE TABLE pinned_track (
+    id                      SERIAL, -- PK
+    user_id                 INTEGER NOT NULL, -- FK to "user".id
+    recording_mbid          UUID NOT NULL,
+    blurb_content           VARCHAR,
+    currently_pinned        BOOLEAN NOT NULL DEFAULT TRUE,
```

Comment:
Let's replace this with "pinned_until TIMESTAMP WITH TIME ZONE NOT NULL". This way when a track is pinned, we set this value to now() + 1 week. Then in the view that displays the user's pinned tracks we can look if the pin has expired and adjust the text accordingly. (e.g. This user has this pinned track X OR This user's last pinned track was X)